### ISSUE TITLE
On the bazel.build site, the standard link has too low contrast. 

### ISSUE THREAD

**Issue by aiuto** [bazelbuild/bazel#9142]
The contrast ratio for links are too low.  

Open Chrome Developer Tools, go to the Audits tab. Run an Audit for bazel.build.

Background and foreground colors do not have a sufficient contrast ratio. 
Low-contrast text is difficult or impossible for many users to read. Learn more.

Failing Elements
--
Documentation<h1 id="documentation">Documentation<a class="anchorjs-link " href="#documentation" aria-label="Anchor link for: documentation" data-anchorjs-icon="" style="font-family: anchorjs-icons; font-style: normal; font-variant: normal; font-weight: normal; line-height: 1; padding-left: 0.375em;"></a></h1>
Edit<a id="gh-edit" class="gh-edit default-hidden" href="https://github.com/bazelbuild/bazel/tree/master/site/docs/install-os-x.md" style="visibility: visible;"><i class="fa fa-pencil" aria-hidden="true"></i> Edit</a>
Use the binary installer (recommended)<a href="#install-with-installer-mac-os-x">Use the binary installer (recommended)</a>
Use Homebrew<a href="#install-on-mac-os-x-homebrew">Use Homebrew</a>
Compile Bazel from source<a href="install-compile-source.html">Compile Bazel from source</a>
bash completion script<a href="completion.html#bash">bash completion script</a>
zsh completion script<a href="completion.html#zsh">zsh completion script</a>
GitHub releases page<a href="https://github.com/bazelbuild/bazel/releases">GitHub releases page</a>
Apple Developer Site<a href="https://developer.apple.com/xcode/downloads/">Apple Developer Site</a>
Bazel releases page on GitHub<a href="https://github.com/bazelbuild/bazel/releases">Bazel releases page on GitHub</a>
PATH<span class="nv">PATH</span>
$PATH<span class="nv">$PATH</span>
$HOME<span class="nv">$HOME</span>
/bin"<span class="s2">/bin"</span>
